auth/tests/unit/cognito-federation-throttles-cloudwatch-alarm.test.ts

```diff
@@ -0,0 +1,227 @@
+import { describe, it, beforeAll, expect } from "vitest";
+import { loadTemplateFromFile } from "../common/template";
+
+const template = loadTemplateFromFile("./template.yaml");
+
+describe("Set up CloudWatch Alarm for Cognito FederationThrottles with supporting alarm resources", () => {
+  let cloudWatchAlarmUnderTest: {
+    Type: any;
+    Properties: any;
+  };
+
+  let snsTopicUnderTest: {
+    Type: any;
+    Properties: any;
+  };
+
+  let subscriptionUnderTest: {
+    Type: any;
+    Properties: any;
+  };
+
+  let topicPolicyUnderTest: {
+    Type: any;
+    Properties: any;
+  };
+
+  beforeAll(() => {
+    const cloudWatchAlarmResources = template.findResources(
+      "AWS::CloudWatch::Alarm"
+    );
```

Comment:
Quick one - is there any reason we do this in the `beforeAll`?

Reply:
Fair comment, removed 👍